core-feature-pack/src/main/resources/content/bin/standalone.bat

```diff
@@ -219,32 +219,44 @@ if "x%JBOSS_CONFIG_DIR%" == "x" (
   set  "JBOSS_CONFIG_DIR=%JBOSS_BASE_DIR%\configuration"
 )
```

Comment:
I checked this again and I don't see these same issues on Windows 7. Here is what I saw https://gist.github.com/jamezp/4da0ef3ab3a80e4d4943b67941bc4ea0.

Reply:
@jamezp I double check this and I confirm my [original comment](https://github.com/wildfly/wildfly-core/pull/3496#discussion_r220894015).

```
VM Flags:
... -XX:MaxHeapSize=2122317824 ...
```

I see this issue on both JDK8 and JDK11, and on both W2012 and W2016. 

Can you please double check? How do you start standalone? Do you use `set "GC_LOG=true"`?

Reply:
@marekkopecky I changed the `standalone.conf.(bat|ps1)` to set the environment variable. Then I just executed `standalone.(bat|ps1)`. This was on both JDK 8 and JDK 11. I checked with `jinfo` and in VisualVM and it seemed to work for me. I didn't get the double quotes like you got around the `JAVA_OPTS` which seems weird. Unfortunately I only have access to Windows 7 which is quite old I realize.

Reply:
@jamezp I check it also on W2008R2 and I see the issue also on W2008. But I'm not sure about W2008r2/W2008 <-> W7 relationship TBH, I guess that W2008 is based on W7.

Anyway, we need to fix this issue.

Reply:
https://en.wikipedia.org/wiki/Windows_Server_2008_R2 says:
`Microsoft introduced Windows Server 2008 R2 at the 2008 Professional Developers Conference as the server variant of Windows 7. `

Reply:
The problem is I'm not sure how to fix this as I only have access to Windows 7. It would be constant guess work and pushing the testing to someone else.

Reply:
I was reminded I can ssh into CI and test there so never mind :)